osu.Game/Screens/Multi/Multiplayer.cs

```diff
@@ -322,43 +302,10 @@ private void subScreenChanged(IScreen lastScreen, IScreen newScreen)
 
             UpdatePollingRate(isIdle.Value);
             createButton.FadeTo(newScreen is LoungeSubScreen ? 1 : 0, 200);
-
-            updateTrack();
         }
 
         protected IScreen CurrentSubScreen => screenStack.CurrentScreen;
 
-        private void updateTrack(ValueChangedEvent<WorkingBeatmap> _ = null)
-        {
-            if (screenStack.CurrentScreen is RoomSubScreen)
-            {
-                var track = Beatmap.Value?.Track;
-
-                if (track != null)
-                {
-                    track.RestartPoint = Beatmap.Value.Metadata.PreviewTime;
-                    track.Looping = true;
-
-                    music?.EnsurePlayingSomething();
```

Comment:
previously, this was run on each beatmap change, ensuring that music starts playing after a download completes, or the multiplayer screen is entered. your changes have removed this call so now it is silent.

Reply:
ah, thought tracks start automatically on beatmap changes, changed.